### PR TITLE
fix JSON parse error Unexpected character (''' (code 39))

### PR DESCRIPTION
**Pull request by malekbenz**
Use an escape character to fix JSON parse error: Unexpected character (''' (code 39))

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@malekbenz Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/gs-accessing-data-rest&pullRequestId=24)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-guides/gs-accessing-data-rest&pullRequestId=24) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@malekbenz Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-guides/gs-accessing-data-rest&pullRequestId=24)!

File: README.adoc [hunk 1]
```diff
@@ -139,7 +139,8 @@ $ curl http://localhost:8080/people
 There are currently no elements and hence no pages. Time to create a new `Person`!
 
 ```
-$ curl -i -X POST -H "Content-Type:application/json" -d '{"firstName": "Frodo", "lastName": "Baggins"}' http://localhost:8080/people
+$ curl -i -X POST -H "Content-Type:application/json" -d "{  \"firstName\" : \"Frodo\",  \"lastName\" : \"Baggins\" }" http://localhost:8080/people
```
Comment (dsyer) [inline README.adoc]: I prefer the original form. If you had a problem what are the steps to reproduce? Is it an OS-specific issue?

BTW the `-X POST` is redundant (unrelated).

**Comment by dsyer** [comment]
I get it, it's a Windows thing. I added a note. Thanks for the suggestion.

**Pull request closed without merging**